src/DataStructures/Variables.hpp

```diff
@@ -48,6 +48,12 @@ class Variables;
  */
 template <typename... Tags>
 class Variables<tmpl::list<Tags...>> {
+ private:
+  // variable template helper for assign_subset
+  template <class T>
+  static constexpr bool in_tags_v =
+      tmpl2::flat_any_v<cpp17::is_same_v<T, Tags>...>;
```

Comment:
I think this is the same as `tmpl::list_contains_v<tmpl::list<Tags...>, T>`, but I'm not sure which is actually clearer with the pack expansions, so I'll leave it up to you whether to change it.